gym/wrappers/order_enforcing.py

```diff
@@ -34,15 +42,10 @@ def reset(self, **kwargs):
         return self.env.reset(**kwargs)
 
     def render(self, **kwargs):
-        """Checks that the environment has been :meth:`reset` before rendering the environment."""
-        if hasattr(self.unwrapped, "disable_render_order_enforcing"):
-            if not self.unwrapped.disable_render_order_enforcing:
-                assert (
-                    self._has_reset
-                ), "Cannot call env.render() before calling env.reset()"
-        else:
-            assert self._has_reset, (
-                "Cannot call env.render() before calling env.reset(), if this is a intended property, "
-                "set `disable_render_order_enforcing=True` on the base environment (env.unwrapped)."
+        """Renders the environment with `kwargs`."""
+        if self._disable_render_order_enforcing is False and not self._has_reset:
```

Comment:
`if not self._disable_render_order_enforcing and not self._has_reset`
Or even
`if not (self._disable... or self._has_reset)`